src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1654,6 +1627,18 @@ public void updateConsumer(@Verify(Consumer.class) String uuid, ConsumerDTO dto)
         guestMigration.buildMigrationManifest(dto, toUpdate);
 
         if (performConsumerUpdates(dto, toUpdate, guestMigration)) {
+            if (toUpdate.checkForCloudIdentifierFacts(dto.getFacts())) {
+                log.warn("Cloud identifier fact change detected");
```

Comment:
Can we also add the Consumer UUID to this warning? It will be helpful for when we are looking in the logs

Reply:
Good idea, done